### DISCUSSION TITLE
Mouse window/screen position missing.

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [microsoft/GDK#65] [Q&A]
Input doesn't currently provide window/screen position for mouse. How do you envision us using this API when we are just using the Windows mouse? I feel like not providing a window position and screen position as well is going to lead people to use the older API's which goes against the purpose of the new Input system. It's just more fragmentation.

Some applications and games will just want a position. I think you should expose this in the mouse state. And change the current mouse state names Position to DeltaSum, because they are not position.

**Comment by bbowman** [comment]
Hey @Gavin-Williams thank you for the suggestion. I have let the feature team know about the discussion here. I can't comment on any roadmap or if the suggestion will be taken at the moment but the right folks are being informed!

Thank you again for helping us improve the GDK!
